WTWSMS/history/titles/c_carcassonne.txt

```diff
@@ -30,20 +30,39 @@
 	holder=190067 #Gaisalaiks
 }
 508.1.1={ 
-	holder=63 #Theodoric 
+	holder=63 #Theodoric
+	liege = "k_italy"
 }
 510.1.1={
 	holder=154003 # Petrus Marcellinus Felix Liberius
-	liege = "k_italy"
 }
 511.1.1={ 
 	holder=190068 #Amalareiks
+	liege = k_visigoths
 }
 # Non Dynastic 
 531.1.1={ 
 	holder=190069 #�iudeis
 }
+548.6.1={ 
+	holder=212861 #Theudegisel
+}
+549.1.1={ 
+	holder=212863 #Agila I 
+}
+554.1.1={ 
+	holder=36555 #Athanagild 
+}
+567.1.1={ 
+	holder=212872 #Liuba
+}
 570.1.1={ 
 	holder=36662 #Gundemar
-	liege = k_visigoths 
+	liege = k_visigoths
```

Comment:
If k_visigoths are the liege from 511.1.1, they don't need to become it again in 570.1.1.

Reply:
It's an unharmful leftover, but I will remove it nonetheless

Reply:
Done